<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBMaterialEditWidget</class>
 <widget class="QWidget" name="SVDBMaterialEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetThermal">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Thermal">
      <attribute name="title">
       <string>Thermal</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelDisplayName">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
       </item>
       <item row="0" column="2">
        <widget class="QtExt::ColorButton" name="pushButtonOpaqueMaterialColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCategory">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Category:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxCategory"/>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelConductivity">
           <property name="text">
            <string>Conductivity [W/mK]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QtExt::ValidatingLineEdit" name="lineEditConductivity"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDensity">
           <property name="text">
            <string>Density [kg/m3]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QtExt::ValidatingLineEdit" name="lineEditDensity"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelSpecHeatCapacity">
           <property name="text">
            <string>Spec. Heat Capacity [J/kgK]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QtExt::ValidatingLineEdit" name="lineEditSpecHeatCapacity"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QGraphicsView" name="graphicsViewCategoryPicture"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Information">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>421</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelDensity_2">
          <property name="text">
           <string>Manfacturer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QtExt::ValidatingLineEdit" name="lineEditDensity_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelDensity_3">
          <property name="text">
           <string>Data Source</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QtExt::ValidatingLineEdit" name="lineEditDensity_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelDensity_4">
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QtExt::ValidatingLineEdit" name="lineEditDensity_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
